### DISCUSSION TITLE
Wildcards Manager dropdown tooltips showing in white text on white background after update to 1.1. Anyone know how to fix this?

### DISCUSSION THREAD

**Discussion by tuangd** [adieyal/sd-dynamic-prompts#414] [Q&A]
This is what happen 
[Image](https://imgur.com/a/b8L5DKB)

Usually when I type __ (that's two underscore), the tooltips will popup and help me sort through the wildcards text files.
Now it's unreadable. 

Anyone know how to fix this?

**Comment by akx** [comment]
I don't think that functionality is provided by this extension at all...
